libraries/WiFi/extra/README.md

```diff
@@ -0,0 +1,16 @@
+# Build scripr for esp-hosted (Portenta C33)
+
+```
+git clone git@github.com:espressif/esp-hosted.git --recursive
+cd esp-hosted/
+patch -p1 < ../0001-Defaults-for-C33-BLE.patch
+cd esp_hosted_fg/esp/esp_driver/
+./esp-idf/install.sh esp32s3
+. esp-idf/export.sh
```

Comment:
```suggestion
. ./esp-idf/export.sh
```

Although functionally equivalent, this command is more friendly because it matches the command printed at the end of the `install.sh` execution of the previous step:

https://github.com/espressif/esp-idf/blob/cbce221e88d52665523093b2b6dd0ebe3f1243f1/install.sh#L28

Reply:
That's not fully equivalent since the file should be "sourced" and not executed. I'd keep the original syntax, or replace with `source esp-idf/export.sh`

Reply:
It is still sourced in my suggestion. Note that the command still starts with the `.` shorthand for `source`.

In my opinion, the `. ./esp-idf/export.sh` command actually makes it easier to understand this confusing double meaning of `.` in the shell in the context here where the previous command appears to have almost the same format at a glance (even though the two are quite different).

>replace with source esp-idf/export.sh

Although I would probably use `.` if I was composing such a command off hand, I personally prefer the less ambiguous `source` in a context like this where a command is written once and used many times (meaning the effort of typing the extra characters is insignificant).

In fact I had originally considered suggesting this change. I reconsidered after I saw that `install.sh` prints the command and uses `.` instead of `source` there. But I still think it would be reasonable to use `source`.

Reply:
ah sorry, I totally misread the suggestion, makes a lot of sense